examples/cryptocurrency-advanced/frontend/src/plugins/blockchain.js

```diff
@@ -4,200 +4,142 @@ import * as proto from '../../proto/stubs.js'
 
 const TRANSACTION_URL = '/api/explorer/v1/transactions'
 const PER_PAGE = 10
-const SERVICE_ID = 128
+const SERVICE_ID = 3
 const TX_TRANSFER_ID = 0
 const TX_ISSUE_ID = 1
 const TX_WALLET_ID = 2
-const TABLE_INDEX = 0
 const Wallet = Exonum.newType(proto.exonum.examples.cryptocurrency_advanced.Wallet)
 
-function TransferTransaction(publicKey) {
-  return Exonum.newTransaction({
-    author: publicKey,
-    service_id: SERVICE_ID,
-    message_id: TX_TRANSFER_ID,
+function TransferTransaction () {
+  return new Exonum.Transaction({
+    serviceId: SERVICE_ID,
+    methodId: TX_TRANSFER_ID,
     schema: proto.exonum.examples.cryptocurrency_advanced.Transfer
   })
 }
 
-function IssueTransaction(publicKey) {
-  return Exonum.newTransaction({
-    author: publicKey,
-    service_id: SERVICE_ID,
-    message_id: TX_ISSUE_ID,
-    schema: proto.exonum.examples.cryptocurrency_advanced.Issue
+function IssueTransaction () {
+  return new Exonum.Transaction({
+    schema: proto.exonum.examples.cryptocurrency_advanced.Issue,
+    serviceId: SERVICE_ID,
+    methodId: TX_ISSUE_ID
   })
 }
 
-function CreateTransaction(publicKey) {
-  return Exonum.newTransaction({
-    author: publicKey,
-    service_id: SERVICE_ID,
-    message_id: TX_WALLET_ID,
-    schema: proto.exonum.examples.cryptocurrency_advanced.CreateWallet
+function WalletTx () {
+  return new Exonum.Transaction({
+    schema: proto.exonum.examples.cryptocurrency_advanced.CreateWallet,
+    serviceId: SERVICE_ID,
+    methodId: TX_WALLET_ID
   })
 }
 
-function getTransaction(transaction, publicKey) {
-  if (transaction.name) {
-    return new CreateTransaction(publicKey)
+function deserializeWalletTx (transaction) {
+  const txTypes = [TransferTransaction(), IssueTransaction()]
+  for (const tx of txTypes) {
+    const txData = tx.deserialize(Exonum.hexadecimalToUint8Array(transaction))
+    if (txData) return Object.assign({}, txData.payload, {
```

Comment:
Nit: What's with this formatting? (Is this some new JS thing? 🙃) I'd suggest to use a block for `if` for it to read better.